src/main/java/org/apache/mesos/scheduler/plan/DefaultBlock.java

```diff
@@ -17,24 +20,23 @@
     private final String name;
     private final Optional<OfferRequirement> offerRequirementOptional;
     private final UUID id = UUID.randomUUID();
+    private final List<String> errors;
+    private final Strategy strategy = new ParallelStrategy();
```

Comment:
Are these assumptions correct:
- Every Element now has a strategy
- A block by default has a a `ParallelStrategy` because we want to launch all the tasks (1 by default) that it composes at once.